src/tests/ftest/dfuse/bash.py

```diff
@@ -126,6 +126,9 @@ def run_bashcmd(self, il_lib=None, compatible_mode=False):
             'fio --readwrite=randwrite --name=test --size="2M" --directory '
             f'{fuse_root_dir}/ --bs=1M --numjobs="4" --ioengine=psync '
             "--group_reporting --exitall_on_error --continue_on_error=none",
+            'fio --readwrite=randwrite --name=test --size="2M" --directory '
+            f'{fuse_root_dir}/ --bs=1M --numjobs="1" --ioengine=libaio --iodepth=4'
+            '--group_reporting --exitall_on_error --continue_on_error=none',
```

Comment:
It would be nicer although not essential to have a dedicated test for aio.

Reply:
Thank you! We could add a dedicated test for fio using libaio either inside PR https://github.com/daos-stack/daos/pull/13797 or a new PR based on this one.